docs/ecu-ElmProtocolTelegramProtocol.js

```
@@ -0,0 +1,286 @@
+   */
+  async handleTelegram(buffer) {
+    this._verifyDependencies();
+
+    if (!buffer || buffer.length === 0) return false;
+
+    if (this.lastTxMsg === buffer) {
+      return false;
+    }
+
+    const responseInfo = this.getResponseId(buffer);
+    const { cleanResponse, id: responseId } = responseInfo;
```

ESLint

[error] 'cleanResponse' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)